### PR TITLE
Devconf us 2024 report

### PR DESCRIPTION
**Pull request by flomedja**

### CODE REVIEW

File: devconf_us_aug_2024/devconf_us_aug_2024.md [hunk 1]
```diff
@@ -0,0 +1,166 @@
...
+
+Some technology professions cannot fully automate their tasks. For instance, **product management** tools have not evolved as quickly as development tools. 
+This can create bottlenecks in delivering enterprise software, potentially leading to unsatisfied customers and lost revenue.
+AI tools can assist **product managers** by analyzing large datasets, automating workflows, and enhancing decision-making capabilities.
+
+Jen highlights the importance of engineering and business teams collaborating to understand each other's processes and agreeing on shared workflows to streamline software delivery.
+Some companies have achieved a change lead time of less than one day. High-performing teams are often the result of investments in development processes. 
+Agile practices, DevOps transformations, and Cloud-Native development are a few examples of processes that help streamline the software development lifecycle.
+
+Finally Jen emphasizes that companies need to empower teams and invest in upskilling.
```
Comment (kinnerabanda) [inline devconf_us_aug_2024/devconf_us_aug_2024.md]: You use past tense and present tense within the summary. 
Eg: "Jen spoke about AI.." vs "Jen highlights the importance of engineering.."

The talk with Kelsey Hightower is written in past tense but the other talks are written in present tense. I would suggest sticking to one tense throughout the article.

File: devconf_us_aug_2024/devconf_us_aug_2024.md [hunk 2]
```diff
@@ -0,0 +1,166 @@
...
+- SRE and platform engineering efforts can be more complex than anticipated. 
+  Live clusters may require 24/7 infrastructure support, monitoring, and observability. Training SREs and platform engineers on the product can take several months.
+- Penetration testing can help identify obvious vulnerabilities before going live.
+- Onboarding teams to new technology takes time. Providing good documentation and code examples can significantly improve the onboarding process.
+
+This talk show the security risk that can happen during the process of delivery a solution to client.
+
+The talk tells the story of the building of [Konflux](https://konflux-ci.dev/) by Red Hat. Konflux help developer builds a secure software supply chain process. It is a spin off of a Entreprise solution devolped at Red Hat.
+
+Ann Marie Fred talk about the challenges encouters during the devolpment of Konflux Enterprise version.
```
Comment (kinnerabanda) [inline devconf_us_aug_2024/devconf_us_aug_2024.md]:
```suggestion
Ann Marie Fred talks about the challenges encoutered during the devolpment of Konflux Enterprise version. 
```

File: devconf_us_aug_2024/devconf_us_aug_2024.md [hunk 3]
```diff
@@ -0,0 +1,166 @@
...
+
+#### Takeaways
+- OCI artifacts are an emerging standard for storing various types of artifacts for container workloads.
+- OCI artifacts can benefit from the same processes and software lifecycle as OCI images.
+
+Andrew Block presents the container runtime specification and explains the different components of a container image.
+
+An OCI image consists of an image manifest, an optional image index, a set of filesystem layers, and a configuration. A container is a running instance of this image. Deep down, containers are files and processes.
+
+The OCI image manifest defines the media type for the image and its layers.
```
Comment (kinnerabanda) [inline devconf_us_aug_2024/devconf_us_aug_2024.md]:
```suggestion
An OCI image consists of an image manifest, an optional image index, a set of filesystem layers, and a configuration. A container is a running instance of this image. Deep down, containers are files and processes. The OCI image manifest defines the media type for the image and its layers.
```

File: devconf_us_aug_2024/devconf_us_aug_2024.md [hunk 4]
```diff
@@ -0,0 +1,166 @@
...
+
+OCI supports storing arbitrary content types called **artifacts**, enabling the storage and serving of content types other than container images.
+
+OCI Artifacts can store:
```
Comment (kinnerabanda) [inline devconf_us_aug_2024/devconf_us_aug_2024.md]:
```suggestion
OCI supports storing arbitrary content types called **artifacts**, enabling the storage and serving of content types other than container images.
OCI Artifacts can store:
```

File: devconf_us_aug_2024/devconf_us_aug_2024.md [hunk 5]
```diff
@@ -0,0 +1,166 @@
...
+- Software Bill Of Materials (SBOM)
+- Signatures
+- Software packages (.jar,rpm)
+- AI/ML models
+
+The benefict of OCI Artifacts :
```
Comment (kinnerabanda) [inline devconf_us_aug_2024/devconf_us_aug_2024.md]:
```suggestion
Benefits of OCI Artifacts:
```

File: devconf_us_aug_2024/devconf_us_aug_2024.md [hunk 6]
```diff
@@ -0,0 +1,166 @@
...
+  It is essential to implement automation and documentation for predictable and reproducible cluster creation.
+- Backups are easy, but recovery can be challenging. It is crucial to perform practical scenarios to ensure that the backup/recovery process works as intended.
+- SRE and platform engineering efforts can be more complex than anticipated. 
+  Live clusters may require 24/7 infrastructure support, monitoring, and observability. Training SREs and platform engineers on the product can take several months.
+- Penetration testing can help identify obvious vulnerabilities before going live.
+- Onboarding teams to new technology takes time. Providing good documentation and code examples can significantly improve the onboarding process.
+
+This talk show the security risk that can happen during the process of delivery a solution to client.
+
+The talk tells the story of the building of [Konflux](https://konflux-ci.dev/) by Red Hat. Konflux help developer builds a secure software supply chain process. It is a spin off of a Entreprise solution devolped at Red Hat.
```
Comment (kinnerabanda) [inline devconf_us_aug_2024/devconf_us_aug_2024.md]:
```suggestion
This talk shows the security risk that can happen during the process of delivery a solution to client. It tells the story of the building of [Konflux](https://konflux-ci.dev/) by Red Hat. Konflux help developer builds a secure software supply chain process. It is a spin off of a Entreprise solution devolped at Red Hat.
```

File: devconf_us_aug_2024/devconf_us_aug_2024.md [hunk 7]
```diff
@@ -0,0 +1,166 @@
...
+- There are four observability signals: logs, metrics, traces, and profiling.
+- Each observability signal contributes to a different aspect of the observability system, and they complement one another.
+- Instrumentation can be either static or dynamic and may vary in scope depending on the system.
+- eBPF can be used to achieve observability without modifying a program's codebase.
+
+Peter Zaitsev presents the use of various observability signals, including logs, metrics, traces, and profiling. 
+He explains the purpose of each signal and how they contribute to system observability.
+
+[eBPF](https://ebpf.io/) allow to get visibility into many Linux kernels events. 
+The talk explains how [eBPF](https://ebpf.io/) programs can be used to have observability without modifying a program's codebase.
```
Comment (kinnerabanda) [inline devconf_us_aug_2024/devconf_us_aug_2024.md]:
```suggestion
[eBPF](https://ebpf.io/) allows visibility into many Linux kernels events. The talk explains how [eBPF](https://ebpf.io/) programs can be used to achieve observability without modifying a program's codebase. 
```

File: devconf_us_aug_2024/devconf_us_aug_2024.md (hunk 1 above)
Comment (flomedja) [reply devconf_us_aug_2024/devconf_us_aug_2024.md]: Fixed :)

File: devconf_us_aug_2024/devconf_us_aug_2024.md (hunk 2 above)
Comment (flomedja) [reply devconf_us_aug_2024/devconf_us_aug_2024.md]: Fixed :)

File: devconf_us_aug_2024/devconf_us_aug_2024.md (hunk 3 above)
Comment (flomedja) [reply devconf_us_aug_2024/devconf_us_aug_2024.md]: Fixed :)

File: devconf_us_aug_2024/devconf_us_aug_2024.md (hunk 4 above)
Comment (flomedja) [reply devconf_us_aug_2024/devconf_us_aug_2024.md]: Fixed :)

File: devconf_us_aug_2024/devconf_us_aug_2024.md (hunk 5 above)
Comment (flomedja) [reply devconf_us_aug_2024/devconf_us_aug_2024.md]: Fixed :)

File: devconf_us_aug_2024/devconf_us_aug_2024.md (hunk 6 above)
Comment (flomedja) [reply devconf_us_aug_2024/devconf_us_aug_2024.md]: Fixed :)

File: devconf_us_aug_2024/devconf_us_aug_2024.md (hunk 7 above)
Comment (flomedja) [reply devconf_us_aug_2024/devconf_us_aug_2024.md]: Fixed :)

**Pull request merged**